tests/integration/test_charm.py

```diff
@@ -107,13 +108,8 @@ async def test_logs_write_to_storage(ops_test: OpsTest):
 
 async def test_exporter_endpoints(ops_test: OpsTest):
     unit_address = await get_address(ops_test=ops_test)
-    node_exporter_url = f"http://{unit_address}:9100/metrics"
     jmx_exporter_url = f"http://{unit_address}:9101/metrics"
```

Comment:
**nitpick/chore:** Use the f-stringed `JMX_EXPORTER_PORT` here.

Reply:
Done in next commit.